src/com/facebook/buck/jvm/kotlin/KotlinLibraryBuilder.java

```diff
@@ -62,7 +62,10 @@ protected CompileToJarStepFactory buildCompileStepFactory() {
       return new KotlincToJarStepFactory(
           Preconditions.checkNotNull(kotlinBuckConfig).getKotlinc(),
           extraKotlincArguments,
-          EMPTY_EXTRA_CLASSPATH);
+          EMPTY_EXTRA_CLASSPATH,
+          getJavac(),
```

Comment:
The tests are complaining javaBuckConfig doesn't exist.  Do you have to do the same thing here that you did for the AndroidKotlinBuilder?